Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Shared/Control Visuals/CommonDialogHandler.cs

```diff
@@ -73,10 +73,10 @@ public CommonDialogHandler(bool embed)
             _embed = embed;
             // Gain access to the global palette
             _manager = new KryptonManager();
-            _backColour = _manager.GlobalPalette.GetBackColor1(PaletteBackStyle.PanelClient, PaletteState.Normal);
-            _defaultFontColour = _manager.GlobalPalette.GetContentShortTextColor1(PaletteContentStyle.LabelNormalPanel, PaletteState.Normal);
-            _inputFontColour = _manager.GlobalPalette.GetContentShortTextColor1(PaletteContentStyle.InputControlStandalone, PaletteState.Normal);
-            _labelFont = _manager.GlobalPalette.GetContentShortTextFont(PaletteContentStyle.LabelNormalPanel, PaletteState.Normal);
+            _backColour = _manager.GlobalCustomPalette!.GetBackColor1(PaletteBackStyle.PanelClient, PaletteState.Normal);
```

Comment:
These should be using the CurrentGlobal_Palette equivalent, nOt custom

Reply:
Added 'ToDo' marker